SparkPost/2web2ui
# FE-291 API Key details view read only

This renders api key details page in read only view. 

Flow:
- (Unlike current behavior) Allows clicks to all links (even api key not owned by current user) 
- The detail page picks up the api key id (and subaccount) from path/query string 
- if the currentUser doesn't match apikey owner, it renders the page in ready only mode
 - all fields read only
 - hides delete button/modal
 - hides submit button

Test:
- create an api key as "normal user"
- create a superuser and login as superuser and go to the api key created in prev step. you should see the form in readonly mode
- rest of the features should work as is. 

Todos


Note: It seemed lot of work for a simple problem (to see grants). We could show it in list like legacy UI. To de-clutter ui, we could show it inline or in modal on a button click.
